superset-frontend/src/components/Menu/SubMenu.tsx

```diff
@@ -42,6 +42,8 @@ const StyledHeader = styled.div`
     padding: 14px 0;
     margin-right: ${({ theme }) => theme.gridUnit * 3}px;
```

Comment:
You can replace all instances of ${({ theme }) => by moving this piece of code to the top like this https://github.com/apache/superset/pull/14448/files#diff-0856ab0ce511fac9256389d5ed3635f15d77ec907de602ce32d28f072de76753

Reply:
Nice tip!! We should sweep through and do this all over the place, on some rainy day (probably not now).